### DISCUSSION TITLE
Custom glb files not get animated

### DISCUSSION THREAD

**Discussion by sunyonglincanada** [visgl/deck.gl#5280] [Q&A] [answered]
Hi,

I am trying to load our own glb file using GLTF loader and ScenegraphLayer, everything works fine using demo glb file, but it did not get animated when using our own glb file. Not sure if this is related to our glb file itself.

Here is the codepen - https://codepen.io/sunyonglincanada/pen/YzGrVmG?editors=1111

Any ideas?

Thanks in advance!!

**Comment by ibgreen** [comment, marked as answer]
The luma.gl glTF implementation only supports the basic node animations. It does not support morph targets, skins etc which are most likely used by your model. 

For most data visualization purposes the current feature set has been sufficient, but it would certainly be nice to offer a full glTF implementation. We have been looking for a contributor  for a while, someone who is interested in improving the glTF support.

**Reply by sunyonglincanada** [reply]
Gotcha, thanks for the updates!!
